examples/benchmarks/README.md

```diff
@@ -107,11 +107,15 @@ python run_all_model.py run 3 lightgbm Alpha158 csi500  # for models with random
 | Model Name | Dataset  | IC          | ICIR        | Rank IC     | Rank ICIR   | Annualized Return | Information Ratio | Max Drawdown |
 |------------|----------|-------------|-------------|-------------|-------------|-------------------|-------------------|--------------|
 | LightGBM   | Alpha158 | 0.0377±0.00 | 0.3860±0.00 | 0.0448±0.00 | 0.4675±0.00 | 0.1151±0.00       | 1.3884±0.00       | -0.0898±0.00 |
+| Transformer| Alpha158 | 0.0245±0.00 | 0.2183±0.04 | 0.0445±0.00 | 0.4324±0.04 | -0.0203±0.02      | -0.2960±0.32      | -0.2189±0.04 |
+| ALSTM      | Alpha158 | 0.0356±0.00 | 0.3351±0.00 | 0.0499±0.00 | 0.4958±0.00 |  0.0531±0.00      | 0.8070±0.00       | -0.1631±0.00 |
```

Comment:
Are you sure that the results of `ALSTM` of different runs nearly do not change? 
This does not  align with the previous results
![image](https://user-images.githubusercontent.com/465606/180918000-d420eb18-549a-453d-974d-fd1497f80c26.png)

Reply:
https://github.com/microsoft/qlib/tree/main/examples/benchmarks#alpha158-dataset